### PR TITLE
Add new cookbook

### PR DESCRIPTION
**Pull request by ornellaaltunyan**
Adding a new cookbook for Unreleased AI

### CODE REVIEW

File: examples/UnreleasedAI/UnreleasedAI.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+To set up streaming and make sure the results are easy to parse, just set `stream` to `true`. The result of the function call is then shown to the user in the frontend of the application.
+
+Running a prompt as an AI function is also a powerful way to automatically set up other Braintrust capabilities. Behind the scenes, Braintrust automatically caches and optimizes the prompt through the [AI proxy](https://www.braintrust.dev/docs/guides/proxy) and logs it to your project, so you can dig into the responses and understand if you need to make any changes. This also makes it easy to change the model in the Braintrust UI, and automatically deploy it to any environment which invokes it.
+
+## Observability
+
+Traditional observability tools monitor performance and pipeline issues, but generative AI projects require deeper insights to ensure your application works as intended. As you continue using the application to generate changelogs for various GitHub repositories, you’ll notice every function call is [logged](https://www.braintrust.dev/docs/guides/logging), so you can examine the input and output of each call. 
+
+![Logs](assets/logs.png)
```
Comment (ankrgyl) [inline examples/UnreleasedAI/UnreleasedAI.md]: Not critical, but we generally make the screenshots in our docs light mode (so they look particularly good on dark mode docs).

If you do change that, I think it'd be useful to make this a GIF or click into one of the logs and show the expanded version. It's nice to see that you can see a richer version of the inputs & outputs.

File: examples/UnreleasedAI/UnreleasedAI.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+![Adding error logs to dataset](assets/add-logs-to-dataset.png)
+
+Alternatively, you can define a dataset in [eval/sampleData.ts](https://github.com/braintrustdata/unreleased-ai/blob/main/eval/sampleData.ts).
+
+Now that we have all three inputs, we can establish an Eval() function in [eval/changelog.eval.ts](https://github.com/braintrustdata/unreleased-ai/blob/6e74be5caed1a1c368ee7124a5adc7e0c27f2969/eval/changelog.eval.ts#L26C1-L36C4):
+
+```typescript
+Eval(PROJECT_NAME, {  
+    data: () => [sampleData],
```
Comment (ankrgyl) [inline examples/UnreleasedAI/UnreleasedAI.md]: should we change this to `initDataset` to reference the newly created dataset? You could do something like:

```
data: initDataset({project: PROJET_NAME, dataset: "My dataset name"}),
//  data: () => [sampleData], // Uncomment to use sample data
```

File: examples/UnreleasedAI/UnreleasedAI.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+`initDataset({project: PROJECT_NAME, dataset: 'eval dataset'})`
+
+Running `pnpm eval` will execute the evaluations defined in [changelog.eval.ts](https://github.com/braintrustdata/unreleased-ai/blob/main/eval/changelog.eval.ts) and log the results to Braintrust.
+
+## Putting it all together
+
+![Developer workflow](assets/developer-workflow.png)
+
+It’s time to [interpret your results](https://www.braintrust.dev/docs/guides/evals/interpret). Examine the comprehensiveness scores and other feedback generated by your evals. Based on these insights, you can make informed decisions on how to improve your application. If the results indicate that your prompt needs adjustment, you can tweak it directly in Braintrust’s UI until it consistently produces high-quality outputs. If tweaking the prompt doesn’t yield the desired results, consider experimenting with different models. You’ll be able to update prompts and models without redeploying your code, so you can make real-time improvements to your product. After making adjustments, re-run your evals to validate the effectiveness of your changes.
```
Comment (ankrgyl) [inline examples/UnreleasedAI/UnreleasedAI.md]: might be nice to have a screenshot of evals?

File: examples/UnreleasedAI/UnreleasedAI.md (hunk 2 above)
Comment (ornellaaltunyan) [reply examples/UnreleasedAI/UnreleasedAI.md]: yeah let me swap it in the code repo

**Review by ankrgyl — APPROVED** [review]
whooo!! let's shipit

**Pull request merged**